src/Chapter20/Listing20.02.AsynchronousWebRequest.cs

```diff
@@ -25,15 +26,16 @@ public static void Main(string[] args)
                 url = args[1];
                 // Ignore additional parameters
             }
-            Console.Write($"Searching for '{findText}' at URL '{url}'.");
+            Console.WriteLine(
+                $"Searching for '{findText}' at URL '{url}'.");
 
             using WebClient webClient = new WebClient();
-            Console.Write("\nDownloading...");
+            Console.Write("Downloading...");
```

Comment:
```suggestion
            Console.WriteLine("Downloading...");
```
Use WriteLine over Write

Reply:
@HannaMichaelis after making this change you will need to make the following other changes too:
Line 38 becomes: `Console.WriteLine("Searching...");
Lines 46-48 become:
```c#
Console.WriteLine(
                        $"'{findText}' appears {
                            textOccurrenceCount} times at URL '{url}'.");
```

Reply:
This is why @Keboo is still the g.o.a.t at reviewing, he catches everything I miss :) Thanks Kevin!